### PR TITLE
View all network call made by the application during text execution(both webview and chrome browser). 

### PR DESCRIPTION
**Pull request by sudharsan-selvaraj**
Currently works only for android. Need to find a way to make it work for ios apps

fix(#36)

https://user-images.githubusercontent.com/20136913/152678168-02231c62-aed5-43dd-abe5-5425d1311462.mp4

.

### CODE REVIEW

File: web/src/components/UI/organisms/session-http-logs.tsx [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    requestTypeFilterId: selectedRequestTypeFilter,
+    textFilter: textFilter,
+  });
+
+  /* Event callbacks */
+  function onRequestTypeFilterChange(index: number) {
+    setSelectedRequestTypeFilter(index);
+  }
+
+  function onTextFilterChange(text: string) {
```
Comment (sbalaji1192) [inline web/src/components/UI/organisms/session-http-logs.tsx:181]: useCallback

File: web/src/store/reducers/entities/logs-reducer.ts [hunk 2]
```diff
@@ -88,6 +94,24 @@ export default createReducer(initialState, {
       isLoading: false,
     },
   }),
+  [ReduxActionTypes.FETCH_SESSION_HTTP_LOG]: (state: LogsState) => ({
+    ...state,
+    http: {
+      ...state.text,
```
Comment (sbalaji1192) [inline web/src/store/reducers/entities/logs-reducer.ts:100]: `...state.http`?

File: web/src/utils/common-utils.ts [hunk 3]
```diff
@@ -112,6 +105,16 @@ export default class CommonUtils {
     return parsedObject;
   }
 
+  public static parseJson(str: any) {
+    if (_.isString(str)) {
+      try {
+        return JSON.stringify(JSON.parse(str), null, 2);
+      } catch (err) {}
+    }
+    console.log(str);
```
Comment (sbalaji1192) [inline web/src/utils/common-utils.ts:114]: remove

File: web/src/components/UI/layouts/parallel-layout.tsx [hunk 4]
```diff
@@ -42,17 +42,43 @@ export function Column(props: ColumnProps) {
...
-const Container = styled.div`
+const Container = styled.div<{
+  responsive?: boolean;
+  responsiveWidth?: number;
+}>`
   display: flex;
   align-items: center;
+
+  ${(props) =>
+    props.responsive &&
```
[the remark this answers is not in the document]
Comment (sudharsan-selvaraj) [reply web/src/components/UI/layouts/parallel-layout.tsx:53]: Yup! For small screens, the layout will collapse into rows
Screenshot 2022-02-10 at 5 08 35 PM

File: web/src/components/UI/organisms/session-build-tab.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  SESSIONS = "Sessions",
+  BUILDS = "Builds",
+}
+
+export default function SesionBuildTab() {
+  return (
+    <Container>
+      <TabsLayout selected={TAB_HEADERS.SESSIONS}>
+        <Tab name={TAB_HEADERS.SESSIONS}>
+          <SessionList />
```
[the remark this answers is not in the document]
Comment (sudharsan-selvaraj) [reply web/src/components/UI/organisms/session-build-tab.tsx:28]: Accidentally committed the file. Will remove it.!

**Pull request merged**